packages/next/client/page-loader.js

```diff
@@ -177,19 +177,27 @@ export default class PageLoader {
   prefetchData(href, asPath) {
     const { pathname: hrefPathname } = parse(href, true)
     const route = normalizeRoute(hrefPathname)
-    return this.promisedSsgManifest.then(
-      (s, _dataHref) =>
-        // Check if the route requires a data file
-        s.has(route) &&
-        // Try to generate data href, noop when falsy
-        (_dataHref = this.getDataHref(href, asPath)) &&
-        // noop when data has already been prefetched (dedupe)
-        !document.querySelector(
-          `link[rel="${relPrefetch}"][href^="${_dataHref}"]`
-        ) &&
-        // Inject the `<link rel=prefetch>` tag for above computed `href`.
-        appendLink(_dataHref, relPrefetch, 'fetch')
-    )
+    if ('requestIdleCallback' in window) {
+      window.requestIdleCallback((deadline) => {
+        if (deadline.timeRemaining() > 0) {
```

Comment:
Seems that this line is not needed: https://github.com/GoogleChromeLabs/quicklink/blob/master/src/index.mjs#L84-L93

Reply:
Updated!!